### PR TITLE
Rename notification event model

### PR DESCRIPTION
**Pull request by nghialv**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/model/BUILD.bazel [hunk 1]
```diff
@@ -53,11 +53,13 @@ go_library(
         "deployment.go",
         "docs.go",
         "environment.go",
-        "event.go",
         "filestore.go",
         "image_name.go",
+        "imagemetadata.pb.go",
+        "imagemetadata.pb.validate.go",
```
Comment (nakabonne) [inline pkg/model/BUILD.bazel:59]: Maybe you should perform `make clean`...

Comment (nghialv) [reply pkg/model/BUILD.bazel:59]: Nice catch!

**Comment by nakabonne** [comment]
Seems that this model is used by only Piped and no communication using it exists. LGTM except for my comment 👍

File: pkg/app/piped/notifier/webhook.go [hunk 2]
```diff
@@ -41,7 +41,7 @@ func (s *webhook) Run(ctx context.Context) error {
 	return nil
 }
 
-func (s *webhook) Notify(event model.Event) {
+func (s *webhook) Notify(event model.NotificationEvent) {
```
[the remark this answers is not in the document]
Comment (khanhtc1202) [reply pkg/app/piped/notifier/webhook.go:44]: use `_` instead 🤔

Comment (nghialv) [inline pkg/app/piped/notifier/webhook.go:44]:
```suggestion
func (s *webhook) Notify(_ model.NotificationEvent) {
```

**Comment by khanhtc1202** [comment]
🚀
/approve

**Pull request merged**